pep-0458.txt

```diff
@@ -337,12 +337,52 @@ paths of available target files together with their cryptographic hashes.
 The file paths must be specified relative to a base URL. This allows the
 actual target files to be served from anywhere, as long as the base URL
 can be accessed by the client. Each top-level role will serve its
-responsibilities without exception.  Figure 1 provides a table of the
+responsibilities without exception.  Table 1 provides an overview of the
 roles used in TUF.
 
-.. image:: pep-0458-1.png
-
-Figure 1: An overview of the TUF roles.
++-----------------------------------------------------------------------------+
+|                          Roles and Responsibilities                         |
++-----------------+-----------------------------------------------------------+
+|       root      | The root roles is the locus of trust for the entire       |
+|                 | repository. The root role signs the root.json metadata    |
+|                 | file. This file indicates which keys are authorized for   |
+|                 | each of the top-level roles, including for the root role  |
+|                 | itself. The roles "root", "snapshot", "timestamp" and     |
+|                 | "targets" must be specified and each has a list of public |
+|                 | keys.                                                     |
++-----------------+-----------------------------------------------------------+
+|     targets     | The targets role is responsible for indicating which      |
+|                 | target files are available from the repository. More      |
+|                 | precisely, the targets role shares the responsibility of  |
+|                 | providing information about the content of updates. The   |
+|                 | targets role may also delegate trust of repository files  |
+|                 | to other roles (delegated roles).                         |
++-----------------+-----------------------------------------------------------+
+| delegated roles | If the top-level targets role performs delegation, the    |
+|                 | resulting delegated roles can then provide their own      |
+|                 | metadata files. The format of the metadata files provided |
+|                 | by delegated targets roles is the same as that of         |
+|                 | targets.json. As with targets.json, the latest version of |
```

Comment:
Might be good to state that the targets role writes to "targets.json", at this point the reader does not know that yet (they might have a suspicion though).

Reply:
Go for it :yolo: